core/.js/src/main/scala/io/udash/utils/FileUploader.scala

```diff
@@ -17,16 +18,15 @@ class FileUploader(url: Url) {
       (0 until input.files.length).map(input.files.item)
     )
 
-  /** Uploads provided `file` in a field named `fieldName`. */
+  /** Uploads provided `file` in a field named `fieldName` with optional custom request headers. */
```

Comment:
very nitty: `custom` -> `additional`, as they don't really have to be custom, they can be well-known and defined headers